armando/style.css

```diff
@@ -152,15 +136,14 @@ we need to adjust them to align them better next to each other. */
 	display: block;
 }
 
-.wp-block-quote,
-.wp-block-quote.has-text-align-right {
-	border-left: none;
-	border-right: none;
-}
-
-.blocks-gallery-caption,
-.comment-body {
-	margin-bottom: 2em;
+/* 
+ * Site logo can not be centered on the front without this.
+ * Temporary solution. Needs testing.
+*/
+.wp-block-site-logo.aligncenter {
```

Comment:
This bug needs to be confirmed, the site logo does not have the same CSS that centers for example the image blocks.

Reply:
I couldn't confirm that bug.  Removing this style I was able to center the site logo (inside of a group block) just fine without it.

Reply:
But is it centered also on the font?